res/values/cm_strings.xml

```diff
@@ -217,6 +217,9 @@
     <string name="battery_maximum_capacity">Maximum capacity</string>
     <string name="battery_maximum_capacity_summary">%1$d mAh (%2$d%%)</string>
     <string name="battery_maximum_capacity_not_available" translatable="false">@string/battery_cycle_count_not_available</string>
+    
+    <!-- Battery Info: mAh -->
+    <integer name="config_battery_divider" translatable="false">1000</integer>
```

Comment:
Move integer to cm_config from cm_strings

Reply:
to cr_strings

Reply:
should i move to cr_string or cr_config?

Reply:
lineage_config.xml

I checked.

There is no cr_config or cr_strings in Settings package.